codewof/programming/content/en/sum-unlucky-7/question.md

```diff
@@ -0,0 +1,7 @@
+# Sum unlucky seven
+
+The following function `sum_unlucky_7(list)` takes a list of integers as an argument, and should **print** the sum of elements in the list.
+However, the number 7 is an unlucky number.
+All 7s should be ignored, and any non-7 numbers immediately after a 7 should also be ignored.
```

Comment:
The sum-lucky-sevens question has been changed so that only the 7's were doubled, rather than the 7's and the number right after them. It would be good to change this question to also be just the 7's, and not the numbers right after them as well.

Reply:
Since it's a debug question rather than a function/parsons I think it's okay to leave it. It becomes very simple if I change it to only focus on the 7s

Reply:
Hmm that's true, I didn't consider that. I'm leaning towards keeping this one the way it is now, but still not 100%
What do you think @lucyturn3r ?

Reply:
I think leaving it is fine - it looks big and scary when looking at the initial code, but the error message they get in the result column should be helpful enough